src/phylum/ci/ci_base.py

```diff
@@ -518,8 +518,8 @@ def check_risk_scores(self, package_result: dict, project_thresholds: dict) -> O
         fail_string += "|-----------|----------|-----|\n"
 
         issue_list = build_issues_list(package_result, issue_flags)
-        for domain, severity, title in issue_list:
-            fail_string += f"|{domain}|{severity}|{title}|\n"
+        for issue in issue_list:
```

Comment:
Changing this from a tuple to a struct seems like a great choice considering a tuple could have easily caused issues here with incorrect ordering since all the fields are strings.